README.md

```diff
@@ -1,6 +1,98 @@
-# dap_job_quality
+# Job Quality Extractor
 
-## Setup
+This project is about extracting dimensions of job quality from online job adverts. This work was funded by the [Economic Statistics Centre of Excellence](https://www.escoe.ac.uk/).
+
+The term "job quality" refers to aspects of a job that affect worker wellbeing - for example how much the job is paid, and whether the contract is permanent. Most research on job quality rightly focuses on data from the employee's point of view, using surveys or interviews or, recently, [online reviews](https://www.escoe.ac.uk/publications/extracting-dimensions-of-job-quality-from-online-employee-reviews/).
+
+Here, we provide a method for identifying dimensions of job quality in online job adverts.
+
+## What dimensions of job quality do you extract?
+
+The term "job quality" refers to aspects of a job that affect worker wellbeing - for example how much the job is paid, and whether the contract is permanent. Most research on job quality rightly focuses on data from the employee's point of view, using surveys or interviews or, recently, [online reviews](https://www.escoe.ac.uk/publications/extracting-dimensions-of-job-quality-from-online-employee-reviews/).
```

Comment:
could delete this here as you have it a few lines up too?

Reply:
Good spot